packages/w3c-webdriver/docs/capabilities.md

```diff
@@ -0,0 +1,58 @@
+# Capabilities
+
+## Capabilities
+
+**PROPERTIES**
+
+- `alwaysMatch?`: [BrowserCapability](#browsercapability)
+- `firstMatch?`: [BrowserCapability](#browsercapability)[]
+
+## BrowserCapability
+
+**PROPERTIES**
+
+- `browserName`: string
+- `goog:chromeOptions?`: [ChromeOptions](#chromeoptions)
+- `moz:firefoxOptions?`: [FirefoxOptions](#firefoxoptions)
+- `se:ieOptions?`: [InternetExplorerOptions](#internetexploreroptions)
+- `bstack:options?`: [BrowserStackOptions](#browserstackoptions)
+
+## ChromeOptions
+
+**PROPERTIES**
+
+- `w3c?`: boolean
+- `binary?`: string
+- `args?`: string[]
+
+## FirefoxOptions
+
+**PROPERTIES**
+
+- `log?`: object
+  - `level?`: string
```

Comment:
Should we worried about these spaces? does it affect the content on the site?

Reply:
level is property of log object. So this is a subitem